lib/Echidna/UI.hs

```diff
@@ -113,17 +113,33 @@ monitor cleanup = let
 
 -- | Heuristic check that we're in a sensible terminal (not a pipe)
 isTerminal :: MonadIO m => m Bool
-isTerminal = liftIO $ queryTerminal (Fd 0)
+isTerminal = liftIO $ (&&) <$> queryTerminal (Fd 0) <*> queryTerminal (Fd 1)
 
 -- | Set up and run an Echidna 'Campaign' while drawing the dashboard, then print 'Campaign' status
 -- once done.
+-- <<<<<<< HEAD
```

Comment:
Please remove all the merge comments (like this one). After that, this PR looks good to merge.

Reply:
should have just been pushed